cloud-aws-common/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsInstanceConnectorTest.java

```diff
@@ -425,6 +431,23 @@ private void testCheckStates(String running, int code, InstanceStatus status) {
         assertThat(result, everyItem(hasProperty("status", is(status))));
     }
 
+    @Test
```

Comment:
Nit: this `@Test` method can be moved up, where the other tests are present

Reply:
Done